test/EmptyApp.FunctionalTests/FunctionalTest/TelemetryModuleWorkingEmptyAppTests.cs

```diff
@@ -7,19 +7,20 @@ public class TelemetryModuleWorkingEmptyAppTests : TelemetryTestsBase
     {
         private const string assemblyName = "EmptyApp.FunctionalTests";
 
-#if NET451
-
         [Fact]
         public void TestBasicDependencyPropertiesAfterRequestingBasicPage()
         {
+#if NET451
```

Comment:
Now it looks like there is a test when there actually is nothing inside it for NetStandard...